gobblin-modules/gobblin-orc/src/main/java/org/apache/gobblin/writer/GobblinBaseOrcWriter.java

```diff
@@ -235,6 +243,9 @@ private synchronized void closeInternal()
       this.flush();
       this.orcFileWriter.close();
       this.closed = true;
+      if (enableRowBatchPool) {
```

Comment:
How often does this occur?

Reply:
As we gradually rollout. This will be important

Reply:
Sorry. was referring to how often we recycle

Reply:
Every 5 mins or 1 min depending upon how quickly we flush